adding speakers to Summit 2025 (#1092)
InnerSourceCommons/innersourcecommons.org: content/en/events/isc-2025.md

</div>

<div class="clearfix speaker-bio">
<img alt="Caroline T Jones photo" src="/images/events/speakers/Summit 2025/Caroline T Jones.jpeg" width="200" class="speaker-photo" id="Caroline+T_Jones" />

**Caroline T Jones** (Kickstart with a Contribfest)

Caroline Jones is an Engineering Manager at The Aerospace Corporation leading a team developing cloud native solutions to tackle some of the most difficult problems in space. She also leads Aerospace’s initiative for software standards and developer best practices, including spearheading the InnerSource project. Caroline began her career at Aerospace as an intern in 2016 before joining full-time in 2017 after completing her B.S. in Computer Engineering at Boston University. She resides in Santa Monica, CA, and has far too many hobbies to list here. 
</div>

<div class="clearfix speaker-bio">

> <img alt="Chamindra de Silva photo" src="/images/events/speakers/Summit 2025/Chamindra de Silva.jpeg" width="200" class="speaker-photo" id="Chamindra_de_Silva" />

Reviewer: ```suggestion
<img alt="Chamindra de Silva photo" src="/images/events/speakers/Summit 2025/Transparent_Square.png" width="200" class="speaker-photo" id="Chamindra_de_Silva" />
```

**Chamindra de Silva** (InnerSource Value Metrics)

> Chamindra de Silva is long time Open Source advocate and contributor originally from Sri Lanka and was active promoting Free and Open Source with FSF and OSI. He has contributions to Apache, Google Summer of Code, UNDP IOSN networks. Open Source projects he has lead have been awarded from SourceForge and Free Software Foundation in the past particularly for his work in the Humanitarian Open Source Domain. He is presently working in Citi in London being the InnerSource Project Manager and Solution Architect for some of the leading InnerSource projects in the organization and is member of the InnerSource governance program. He has published articles and papers in CACM, IEEE, IDRC, UNDP, UN ESCAP and CMI. ( I will be presenting with Daniel Izquierdo Cortázar)

Reviewer: ```suggestion
Chamindra de Silva is long time Open Source advocate and contributor originally from Sri Lanka and was active promoting Free and Open Source with FSF and OSI. He has contributions to Apache, Google Summer of Code, UNDP IOSN networks. Open Source projects he has lead have been awarded from SourceForge and Free Software Foundation in the past particularly for his work in the Humanitarian Open Source Domain. He is presently working in Citi in London being the InnerSource Project Manager and Solution Architect for some of the leading InnerSource projects in the organization and is member of the InnerSource governance program. He has published articles and papers in CACM, IEEE, IDRC, UNDP, UN ESCAP and CMI.
```

Reviewer: Don't need the note about Daniel.

</div>

<div class="clearfix speaker-bio">

> <img alt="Clare Dillon photo" src="/images/events/speakers/Summit 2025/Clare Dillon.jpeg" width="200" class="speaker-photo" id="Clare_Dillon" />

Reviewer: This photo doesn't exist, yet, so it looks broken when the website is trying to display.

<img width="1366" height="768" alt="Screenshot 2025-09-25 2 15 35 PM" src="https://github.com/user-attachments/assets/e3d2d307-3664-44fd-b746-e075c65968a9" />

Need to just have no photo (e.g. white square) or silhouette photo until you get the real ones.

**Gilles Gravier** (InnerSource in Government)

Gilles is a director, and senior open source, InnerSource and innovation strategy advisor. Based in Geneva, Switzerland, he provides innovation strategy consulting and advisory services to his key customers worldwide, through the application of open source, InnerSource, blockchain, metaverse, quantum and other highly innovative technologies. He can also offer service as a Chief Open Source or InnerSource Officer, or Head of Innovation for your company. After delivering these services for almost 10 years as part of Wipro's open source consulting practice, he is now proposing his skills and experience as an independent consultant.
He is currently working for the State Administration of Geneva’s IT Services (OCSIN) where, after creating their open source strategy, he is driving its implementation as open source and InnerSource programs.
</div>
<div class="clearfix speaker-bio">
<img alt="Isabel Drost-Fromm photo" src="/images/events/speakers/Summit 2025/Isabel Drost Fromm.jpg" width="200" class="speaker-photo" id="Isabel_Drost_Fromm" />

**Isabel Drost-Fromm** (Open communication for open development)

> Isabel Drost-Fromm was up to recently the Chair of the board of directors of the InnerSource Commons Foundation, as well as (former board) member of the Apache Software Foundation. Interested in all things search and text mining with a thorough background in open source collaboration, she is working at Europace AG as Open Source Strategist. True to the nature of people living in Berlin she loves giving friends a reason for a brief visit - as a result she co-founded and is still one of the creative heads behind Berlin Buzzwords, a tech conference on all things search, scale and storage and FOSS Backstage.

Reviewer: ```suggestion
Isabel Drost-Fromm recently finished services as the Chair of the board of directors of the InnerSource Commons Foundation, as well as (former board) member of the Apache Software Foundation. Interested in all things search and text mining with a thorough background in open source collaboration, she is working at Europace AG as Open Source Strategist. True to the nature of people living in Berlin she loves giving friends a reason for a brief visit - as a result she co-founded and is still one of the creative heads behind Berlin Buzzwords, a tech conference on all things search, scale and storage and FOSS Backstage.
```

</div>

<div class="clearfix speaker-bio">
<img alt="Jeff Bailey photo" src="/images/events/speakers/Summit 2025/Jeff Bailey.png" width="200" class="speaker-photo" id="Jeff_Bailey" />

**Jeff Bailey** (Effective InnerSource Strategies for Resource-Constrained Teams)

Jeff is a software development leader at Nike with 25+ years of experience building full-stack applications on numerous platforms to solve business problems. Jeff applies knowledge of several programming languages to deliver high-quality software solutions. With a continuous improvement mindset he focuses on leading software product development communities, boosting productivity, and automating almost everything. His work at Nike revolves around growing Communities of Practice, InnerSource, and evangelizing global adoption of Platform Engineering to drive efficiency.
</div>

<div class="clearfix speaker-bio">
<img alt="Jerry Tan photo" src="/images/events/speakers/Summit 2025/Jerry Tan.jpeg" width="200" class="speaker-photo" id="Jerry_Tan" />

**Jerry Tan** (AI for InnerSource & InnerSource For AI)

> Open Source Expert,  Board member of InnerSourceCommons,  now serves as Executive Vice Secretary-General of COPU(China Open Source Promotion Union ),  

Reviewer: ```suggestion
Open Source Expert,  Board member of InnerSourceCommons,  now serves as Executive Vice Secretary-General of COPU (China Open Source Promotion Union),  
```

**Maximilian Capraro** (Applying InnerSource for DATEV's Shared Frontend Library)

Dr. Maximilian Capraro is software engineer at DATEV eG where he consults on InnerSource and open source and is maintainer in one of the firm's most successful InnerSource projects. Max is a co-founding member of the InnerSource Commons Foundation and served two terms on its board of directors. Back in academia, Max performed over six years of research, published multiple articles on InnerSource in peer reviewed venues, and consulted companies including Siemens, Continental, and Black Duck Software. He developed the contribution-flow method for evaluating and auditing InnerSource success in large organizations. His interests include InnerSource governance, transfer pricing for InnerSource, as well as community analytics and management. Max holds a doctoral degree from FAU Erlangen, Germany. Learn more about Max at capraro.net.
</div>

<div class="clearfix speaker-bio">
<img alt="Micaela D Eller photo" src="/images/events/speakers/Summit 2025/Micaela.jpeg" width="200" class="speaker-photo" id="Micaela_D_Eller" />

> **Micaela D Eller** (Is the Future Is Forked? Navigating an Uncertain Era via InnerSource)(We are People not Robots: An IBM InnerSource Story)

Reviewer: ```suggestion
**Micaela D Eller** (Is the Future Forked? Navigating an Uncertain Era via InnerSource) (We are People not Robots: An IBM InnerSource Story)
```

</div>

<div class="clearfix speaker-bio">
<img alt="Yuki Hattori photo" src="/images/events/speakers/Summit 2025/Yuki Hattori.jpg" width="200" class="speaker-photo" id="Yuki_Hattori" />

**Yuki Hattori** (Navigating the AI Code Explosion: InnerSource Strategies for Quality at Scale)

Yuki Hattori is a Senior Architect at GitHub, renowned for his expertise in cloud technologies, DevOps methodologies, and AI-driven software development. He serves concurrently as President of the InnerSource Commons Foundation, leading global efforts to promote the adoption of InnerSource practices that break organizational silos and enhance collaboration.
At GitHub, Yuki accelerates enterprise adoption of GitHub Copilot and integrates open-source methodologies within corporate environments to boost efficiency and innovation. Previously at Microsoft, he spent seven years driving Azure cloud adoption and DevOps in mission-critical manufacturing systems.
</div>

<div class="clearfix speaker-bio">
<img alt="Zachery Koppert photo" src="/images/events/speakers/Summit 2025/zack_koppert.jpeg" width="200" class="speaker-photo" id="Zachery_Koppert" />

> **Zachery Koppert** (Can you measure InnerSource?)

Reviewer: ```suggestion
**Zachery Koppert** (Can you Measure InnerSource?)
```
